src/test/level1.rs

```diff
@@ -17,24 +17,24 @@ fn power_test() {
             (
                 r"2 ^ {3.2}",
                 Expr::Operation {
-                    l: Box::new(Expr::Int(2)),
-                    r: Box::new(Expr::Float(3.2)),
+                    l: Box::new(Expr::Value(2.)),
```

Comment:
为什么是2.

Reply:
Because I integrated Int and Float to Value, and the type of it is f64. Whether you input 2 or 2.0, ValueParser will parser it into `2.0`. In Rust, type i64 couldn't be convert to f64 automatically, so you should pass f64 into here.